### PR TITLE
[14.0] [MIG] stock_orderpoint_uom

### PR DESCRIPTION
**Pull request by DavidBForgeFlow**
Standard migration to v14.

### CODE REVIEW

File: stock_orderpoint_uom/models/procurement_group.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ProcurementGroup(models.Model):
+    _inherit = "procurement.group"
+
+    @api.model
+    def run(self, procurements):
```
Comment (MiquelRForgeFlow) [inline stock_orderpoint_uom/models/procurement_group.py]:
```suggestion
    def run(self, procurements, raise_user_error=True):
```
run is a function of core

File: stock_orderpoint_uom/models/procurement_group.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                            procurement.values,
+                        )
+                    )
+                    indexes_to_pop.append(i)
+        if new_procs:
+            indexes_to_pop.reverse()
+            for index in indexes_to_pop:
+                procurements.pop(index)
+        procurements.extend(new_procs)
+        return super(ProcurementGroup, self).run(procurements)
```
Comment (MiquelRForgeFlow) [inline stock_orderpoint_uom/models/procurement_group.py]:
```suggestion
        return super(ProcurementGroup, self).run(procurements, raise_user_error)
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by mariadforgeflow — APPROVED** [review]
LGTM

**Pull request merged**